lib/kramdown-rfc2629.rb

```diff
@@ -495,8 +501,10 @@ def convert_img(el, indent, opts) # misuse the tag!
           end
         end
         if alt == ":include:"   # Really bad misuse of tag...
+          label = el.attr.delete('label')
+          warn "*** no label for anchor #{anchor}" unless label
           to_insert = ""
-          anchor.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z0-9.-]+)/) do |t, n|
+          anchor.scan(/(W3C|3GPP|[A-Z-]+)[.]?([A-Za-z0-9.\/-]+)/) do |t, n|
```

Comment:
Yeah, I note an example on xml2rfc.tools.ietf.org that uses `+`, but I didn't add that.  I'll let you do that part.